src/components/play-pause/play-pause.tsx

```diff
@@ -63,6 +63,8 @@ class PlayPause extends Component<any, any> {
         this._playPauseButtonRef?.focus();
       });
     });
+    const myPlayerContainer = document.querySelector('.kaltura-player-container');
```

Comment:
there might be more than one instance of Kaltura-player on page. You have to specify exact ID of player in selector. Id of player can be get from player configuration (targetId)